assets/styles/base/_typography.scss

```diff
@@ -37,10 +37,43 @@ H6 {
 }
 
 P {
-  font-size: 1.4em;
+  // font-size: 1.4em;
   line-height: 1.75em;
   font-weight: 400;
   font-style: normal;
   letter-spacing: 0em;
   margin: 0;
 }
+
+UL {
+  LI {
+    margin: 10px 0;
+  }
+}
```

Comment:
You really want this on every LI everywhere by default?  i.e. the left nav spacing seems to be getting out of hand..

![image](https://user-images.githubusercontent.com/753917/97228580-5e529080-1794-11eb-8540-f0902aeced5f.png)